test/DeltaTriplesTest.cpp

```diff
@@ -0,0 +1,269 @@
+// Copyright 2023 - 2024, University of Freiburg
+//  Chair of Algorithms and Data Structures.
+//  Authors:
+//    2023 Hannah Bast <bast@cs.uni-freiburg.de>
+//    2024 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
+
+#include <gtest/gtest.h>
+
+#include "./util/GTestHelpers.h"
+#include "./util/IndexTestHelpers.h"
+#include "absl/strings/str_split.h"
+#include "engine/ExportQueryExecutionTrees.h"
+#include "index/DeltaTriples.h"
+#include "index/IndexImpl.h"
+#include "index/Permutation.h"
+#include "parser/RdfParser.h"
+
+namespace Matchers {
```

Comment:
Use the anonymous namespace for such helpers.
If you want, you can still use a named namespace inside the anonymous namespace
`namespace { namespace Matchers {`.